collectives/models/configuration.py

```diff
@@ -148,7 +148,7 @@ class ConfigurationItem(db.Model):
 
     :type: int"""
 
-    date = db.Column(db.DateTime, server_default=func.now())
+    date = db.Column(db.DateTime, server_default=func.now())  # pylint: disable=E1102
```

Comment:
Pour simplifier la lecture, c'est mieux d'utiliser le nom complet , ici `disable=not-callable`

Reply:
Bon en fait j'ai essayé, mais Blake fails lorsque je n'explicite pas l'id de l'erreur.... J'ai précisé la signification de l'erreur à côté

Reply:
My bad, j'avais fait une typo